RocketCast/PodcastModel.swift

```diff
@@ -8,15 +8,17 @@
 
 import Foundation
 
-
-class PodcastModel {
-    private var xml: String?
+struct PodcastModel {
     
-    //podcast info
-    //podcast episodes
+    var title = NSMutableString()
+    var author = NSMutableString()
+    var description = NSMutableString()
+    var episodes = [EpisodeModel]()
+    var imageURL = ImageWebURL()
```

Comment:
This is pretty much the same as doing `vat imageURL = String()` which compiles but does not feel right